packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -151,6 +128,21 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
             }
         }
     }
+
+    public checkDuplicateLabel(newFullPath: string, nodesToCheck: IZoweUSSTreeNode[], newNodeType: string) {
+        for (const node of nodesToCheck) {
+            const nodeType = contextually.isFolder(node) ? "folder" : "file";
+            if (newFullPath === node.fullPath.trim()) {
+                return localize(
+                    "renameUSSNode.duplicateName",
+                    "A {0} already exists with this name. Please choose a different one.",
```

Comment:
@katelynienaber This is suuuper nitpicky, but could we please say, "Please choose a different name?"

I think it makes the instruction a bit clearer.

Reply:
I don't like reusing the word "name" twice in a row, but I guess clarity is more important than literary grace here XD

Fixed.